packages/blade/src/components/List/listTokens.ts

```diff
@@ -0,0 +1,152 @@
+import type { CodeProps } from '../Typography';
+import type { ListProps } from './List';
+import type { TypographyPlatforms } from '~tokens/global/typography';
+import type { DotNotationSpacingStringToken } from '~src/_helpers/types';
+
+const listItemPaddingLeft: Record<
```

Comment:
do we really need to have padding? can we not use margins here?

Reply:
Made it margins. I wanted to do this earlier but lost track of it